### ISSUE TITLE
logits' dimension

### ISSUE THREAD

**Issue by hanna-kim** [fabianbormann/Tensorflow_Segmentation#31]
Hi?
I am currently working to make PR of DeconvNetPipeline.py with batch normalization.
I wonder, however, why the logits' last dimension is 21 at the end of the network, while ?
Does self.y has same 21 at final dimension? 

Thank you

**Comment by fabianbormann** [comment]
closed by e4d59e9 (see new batch generator method)
